readme.md

```diff
@@ -0,0 +1 @@
+goddamnit RJ
```

Comment:
I think that this would be better "Thou hast been Fuched"

Reply:
Thanks for the feedback Timothy!